### ISSUE TITLE
Improve CSS Accessibility Quiz question set

### ISSUE THREAD

**Issue by huyenltnguyen** [freeCodeCamp/freeCodeCamp#58090]
## Description

I just quickly went through the question set of the CSS Accessibility Quiz, and I think most / a lot of questions in there should be replaced.

<details>
  <summary>Question list</summary>

Yeah, I lied. By "improve", I meant rewrite :(

- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L20
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L42
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L64
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L86
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L108
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L152
  - We probably can keep this since it's an a11y question, but I think it's more appropriate in an HTML quiz rather than a CSS one.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L174
  - We probably can keep this since it's an a11y question, but I think it's too trivial at this point of the curriculum (campers should already know what a screen reader is from the HTML chapter).
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L196
  - The question isn't relevant to the quiz' focus area.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md?#L218
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L240
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L306
  - The question isn't relevant to the quiz' focus area.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/6ac3763b81d1a1c4d6a21416a8625ff9a9d96cb8/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L328
  - This question is already covered in the HTML chapter ([672a54a6675c168faa84252d.md](https://github.com/freeCodeCamp/freeCodeCamp/blob/6ac3763b81d1a1c4d6a21416a8625ff9a9d96cb8/curriculum/challenges/english/25-front-end-development/lecture-importance-of-accessibility-and-good-html-structure/672a54a6675c168faa84252d.md#L17)), and it should stay there when the a11y concept is first introduced.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md#L350
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md?plain=1#L394
  - I guess making a web page responsive could count as a11y (users who can't afford a massive device can still view and use the page). But I think the question itself is too vague for campers to pick the correct answer.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md?plain=1#L416
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.
- https://github.com/freeCodeCamp/freeCodeCamp/blob/a2eedb7c0c551d9ddaf511f6faf793aa2d423028/curriculum/challenges/english/25-front-end-development/quiz-css-accessibility/66ed8fc1f45ce3ece4053ead.md?plain=1#L438
  - The question isn't relevant to the quiz' focus area, and it's also too trivial at this point of the curriculum.

---

</details>

IMVHO, I think part of this issue comes from the lack of content in the module (and thus we don't have much topics to quiz on).

[The lecture](https://www.freecodecamp.org/learn/full-stack-developer/#lecture-best-practices-for-accessibility-and-css) only touches on color contrast checker tools and how to visually hide elements, while I think it should cover a11y techniques like those introduced in the [Build a Quiz Webpage](https://www.freecodecamp.org/learn/full-stack-developer/#workshop-accessibility-quiz) as well.

Also, I think we should have a lecture video explaining why sufficient color contrast is important, and it should come before [What Are Some Tools to Check for Good Color Contrast on Sites?](https://www.freecodecamp.org/learn/full-stack-developer/lecture-best-practices-for-accessibility-and-css/what-are-some-tools-to-check-for-good-color-contrast-on-sites).

**Comment by moT01** [comment]
Wonder if we should shorten this to 10 questions or something - or combine it with another short module. There really isn't much content.

**Comment by moT01** [comment]
I think I would prefer to keep the chapters how they are. I think it's fine that the questions don't entirely match the name of the challenge/block/module - I think I would consider renaming it before splitting it into it's own chapter.

**Comment by jdwilkin4** [comment]
The functionality for 10 questions was merged in, so we could set this quiz to just 10 👍
